core/store/src/trie/mem/updating.rs

```diff
@@ -13,13 +13,12 @@ use super::metrics::MEM_TRIE_NUM_NODES_CREATED_FROM_UPDATES;
 use super::node::{InputMemTrieNode, MemTrieNodeId, MemTrieNodeView};
 use crate::trie::insert_delete::NodesStorage;
 use crate::trie::{
-    Children, MemTrieChanges, NodeHandle, StorageHandle, StorageValueHandle, TrieNode,
-    TrieNodeWithSize, TrieRefcountDeltaMap, ValueHandle, TRIE_COSTS,
+    Children, MemTrieChanges, StorageValueHandle, TrieRefcountDeltaMap, ValueHandle, TRIE_COSTS,
 };
 use crate::{NibbleSlice, RawTrieNode, RawTrieNodeWithSize, TrieChanges};
 use near_primitives::errors::StorageError;
 use near_primitives::hash::{hash, CryptoHash};
-use near_primitives::state::{FlatStateValue, GenericTrieValue};
+use near_primitives::state::{FlatStateValue, GenericTrieValue, ValueRef};
 use std::collections::{BTreeMap, HashMap};
 use std::sync::Arc;
 
```

Comment:
Line 107 comment, does it still apply?
```
    /// TODO(#12324): replace `TrieNode::memory_usage_direct_internal` and
    ///`MemTrieNodeView::memory_usage` by this method.
```

Reply:
Yes, because TrieNode is still used. Moved TODO to function which should be removed.